smug.go

```diff
@@ -57,6 +59,70 @@ func (smug Smug) switchOrAttach(sessionName string, attach bool, insideTmuxSessi
 	return nil
 }
 
+//Exists verifies if file exists
+func Exists(path string) (bool, error) {
+	_, err := os.Stat(path)
+	if err == nil {
+		return true, nil
+	}
+	if os.IsNotExist(err) {
+		return false, nil
+	}
+	return false, err
+}
+
+//ConfigPath returns a string of the full filepath
+func ConfigPath(options Options) string {
```

Comment:
let's remove this method at all, and instead inject `configPath` to the `Smug` struct, e.g:
```go
type Smug struct {
    tmux Tmux
    commander Commander
    configPath string
}
```